packages/api/src/external/carequality/patient-shared.ts

```diff
@@ -1,11 +1,37 @@
-import { PatientExternalDataEntry } from "@metriport/core/domain/patient";
+import { PatientExternalDataEntry, PatientDemoData } from "@metriport/core/domain/patient";
+import { LinkStatus } from "../patient-link";
+import { ScheuledPatientDiscovery } from "../hie/schedule-patient-discovery";
 
 export class PatientDataCarequality extends PatientExternalDataEntry {
   constructor(
     /**
      * The status of the patient discovery.
      */
-    public discoveryStatus?: "processing" | "completed" | "failed",
+    public discoveryStatus?: LinkStatus | undefined,
+    /**
+     * The request ID of the patient discovery.
+     */
+    public discoveryRequestId?: string,
+    /**
+     * The facility ID of the patient discovery.
+     */
+    public discoveryFacilityId?: string,
+    /**
+     * The start date of the patient discovery.
+     */
+    public discoveryStartedAt?: Date,
+    /**
+     * The flag for determining whether to re-run patient discovery once if new demographic data is found.
+     */
+    public rerunPdOnNewDemographics?: boolean,
+    /**
+     * The most recent payload used for patient demographics augmetnation.
+     */
+    public augmentedDemographics?: PatientDemoData,
+    /**
+     * The request ID for the next patient discovery triggered while the patient discovery was processing.
+     */
+    public scheduledPdRequest?: ScheuledPatientDiscovery,
```

Comment:
scheduledPdRequestId ?

Reply:
Was a typo leftover from a version where I had just an ID, not a full payload. Fixed.